slides/week7/week7.org

```diff
@@ -0,0 +1,55 @@
+#+TITLE: Week 7
+#+AUTHOR: Josh Ting 
+#+EMAIL: josh.ting@gatech.edu
+
+* Data Structures
+- Data structures are ways that computers store collections of data in memory
+- Different data structures offer different advantages and disadvantages in terms of how data is grouped or how efficient it is to interact with the data
+
+* Arrays
```

Comment:
I don't like that we're mixing concrete implementations (array, linked list) with container abstractions (list, set, map). 

I think that you should define an abstraction and possible implementations of it, to keep the difference clear.